### DISCUSSION TITLE
How to configure a non-standard git port

### DISCUSSION THREAD

**Discussion by roland-d** [WeblateOrg/weblate#10646] [Q&A] [answered]
Hello all,

I am in the process of setting up a self-hosted Weblate instance and buzzing along just fine until now :) I am trying to setup a translation component which needs a URL to my git repo. My git repo is a self-hosted Forgejo/Gitea instance but it runs on port 222 instead of 22 due to it being in use.

Now I can´t seem to find anything on how to configure this port somewhere. The git:// URL does not allow for a port and a git config file cannot be created as it is a read-only filesystem.

I have searched the Weblate documentation, tried the search engines, tried the issues and tried the discussions but can't seem to find anything related tot this. Am I missing the obvious here?

Thank you for your help.

**Comment by nijel** [comment, marked as answer]
`git://` URL is for the legacy Git protocol, and you probably don't want to use that because it is unencrypted.

Instead, use `ssh://` URL, which definitely works with a port specified in the URL.

**Reply by roland-d** [reply]
Thank you very much.
